scss/components/breadcrumb.scss

```diff
@@ -66,4 +66,27 @@ $block: #{$fd-namespace}-breadcrumb;
             margin-left: $fd-breadcrumb-link-margin;
         }
     }
+
+    a {
+        text-decoration: none;
+        display: inline-block;
+        transition: all $fd-animation--speed ease-in;
+        @include fd-var-color("color", $fd-color--action, --fd-color-action);
+        @include fd-hover() {
+          @include fd-var-color("color", fd-color-state("hover", "action"), --fd-color-action-hover);
+        }
+        @include fd-active-pressed-selected() {
+          @include fd-var-color("color", fd-color-state("selected", "action"), --fd-color-action-selected);
+          outline: none;
+        }
+        &:focus {
+          outline-style: dotted;
+          outline-width: 1px;
+          @include fd-var-color("outline-color", fd-color-state("hover", "action"), --fd-color-action-focus);
+        }
+        @include fd-disabled() {
+          @include fd-var-color("outline-color", fd-color-state("disabled", "action"), --fd-color-action-disabled);
+          cursor: not-allowed;
+        }
+    }
```

Comment:
This block needs to be changed because it's tied directly to an element (`<a>` in this case).  Although it's a child of the entire block class, styles can bleed in or out of this without a class declaration.  Additionally, are all these really needed?  Shouldn't the links just be using the `.fd-link` class anyway and then offer minor adjustments to it?